cl/recap/index/index_recap_dockets.sh

```diff
@@ -0,0 +1,3 @@
+
+cd ../../../
+python manage.py cl_update_index --update --source recap --type dockets --solr-url http://127.0.0.1:8983/solr/dockets
```

Comment:
This file is unnecessary. Re-indexing is something we do pretty regularly, for better or worse. You can also remove `__init__.py`, I believe.